### ISSUE TITLE
Codeblocks files are gone

### ISSUE THREAD

**Issue by SynchromaAE** [openframeworks/ofBook#297]
In every version of OF from 0.9.0 onwards, the Codeblocks workspace files (.workspace) or project files (.cbp) are not included. The book needs to be updated to reflect this, possibly with a "Making a new OF project in Codeblocks" page. I am a beginner with OF and Codeblocks, but not with coding, and this is a serious omission for beginners.

**Comment by hamoid** [comment]
It's here: https://openframeworks.cc/ofBook/chapters/setup_and_project_structure.html

I created a screenshot for the book. Although I'm not sure what to write about it... 

#### QtCreator

[QtCreator](https://www.qt.io/qt-features-libraries-apis-tools-and-ide/#ide) is a cross platform free/libre IDE. It is quite "light" in terms of downloading but powerful nonetheless.
![qtcreator-of](https://user-images.githubusercontent.com/108264/50044536-3157d180-0085-11e9-873a-e39a1a48b5f6.png)

**Comment by roymacdonald** [comment]
Hi! @SynchromaAE Thanks for pointing this out. 
I will take care of this.
@hamoid Thanks for the screen shot! :)

Cheers

**Comment by hamoid** [comment]
Welcome :) BTW, there's more mentions of code::Blocks.
See https://github.com/openframeworks/ofBook/search?utf8=%E2%9C%93&q=%22code%3A%3Ablocks%22&type=

**Comment by hamoid** [comment]
@roymacdonald Based on https://github.com/qt-creator/qt-creator/blob/master/LICENSE.GPL3-EXCEPT it's GPL3 with 2 exceptions.

**Comment by arturoc** [comment]
@roymacdonald perhaps it's easier to just remove any mention of any particular IDE and just mention that the guides to install each IDE on each platform are in the downloads section. Even the VS reference seems super outdated now

That way whenever we add new IDEs or just update versions there's no outdated instructions

You could include mentions of well known IDEs but  making very clear that are just examples, not mentioning any specific version and linking to the install guides so there's no confusions like this

**Comment by roymacdonald** [comment]
@arturoc yeah. you're right. I was going to include the links to each setup guide but it is much better as you say. I think that it is still a good idea to keep the screenshots as these make sense with the chapter's logic. 
cheers

**Comment by roymacdonald** [comment]
Hi @SynchromaAE , @arturoc , @hamoid 
I just made a lot of improvements. Please check this PR. https://github.com/openframeworks/ofBook/pull/298
Some proof reading will be welcome. 
cheers
